addons/stock/views/product_strategy_views.xml

```diff
@@ -1,51 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="view_putaway" model="ir.ui.view">
-        <field name="name">product.putaway.form</field>
+    <record id="stock_putaway_list" model="ir.ui.view">
+        <field name="name">product.putaway.tree</field>
         <field name="model">product.putaway</field>
         <field name="arch" type="xml">
-            <form string="Putaway">
-                <group colspan="4">
-                    <field name="name"/>
-                </group>
-                <p class="oe_grey">
-                The rules defined per product will be applied before the rules defined per product category.
-                </p>
-                <field name="product_location_ids" colspan="4" nolabel="1">
-                    <tree editable="bottom">
-                        <field name="sequence" widget='handle'/>
-                        <field name="product_id" required="1"/>
-                        <field name="fixed_location_id" domain="[('id', 'child_of', context.get('location_id'))]"/>
-                    </tree>
-                </field>
-                <field name="fixed_location_ids" colspan="4" nolabel="1">
-                    <tree editable="bottom">
-                        <field name="sequence" widget='handle'/>
-                        <field name="category_id" required="1"/>
-                        <field name="fixed_location_id" domain="[('id', 'child_of', context.get('location_id'))]"/>
-                    </tree>
-                </field>
-            </form>
-        </field>
-    </record>
-
-    <record id="stock_fixed_putaway_strat_form" model="ir.ui.view">
-        <field name="name">stock.fixed.putaway.strat.form</field>
-        <field name="model">stock.fixed.putaway.strat</field>
-        <field name="arch" type="xml">
-            <form string="Putaway">
-                <group>
-                    <group>
-                        <field name="putaway_id"/>
-                        <field name="fixed_location_id"/>
-                        <field name="sequence"/>
-                    </group>
-                    <group>
-                        <field name="category_id"/>
-                        <field name="product_id"/>
-                  </group>
-                </group>
-            </form>
+            <tree string="Putaway Rules" editable="bottom">
+                <field name="sequence" widget="handle"/>
```

Comment:
i don't want the sequence widget if we're coming from the product template stat button because ordering there does not make sense (mixed rule on product and categories)

invisible=context.get and forge the context in the stat button